installer/common/install-operators.sh

```diff
@@ -85,12 +85,12 @@ if [ "$installSparkOperator" = true ]; then
     --set operatorVersion="$sparkOperatorImageVersion" \
     incubator/sparkoperator)
 
-    if [ $? -ne 0 ]; then 
+    if [ $? -ne 0 ]; then
         print_error_message "$result"
         print_error_message "installation failed"
         exit 1
     fi
 
     # Label the deployment so we can detect if we installed it later
-    kubectl label deployment -n "$sparkOperatorNamespace" "$sparkOperatorReleaseName"-sparkoperator installed-by=cloudflow --overwrite
+    kubectl label deployment -n "$sparkOperatorNamespace" "$sparkOperatorReleaseName" installed-by=cloudflow --overwrite
```

Comment:
Aligns with ```$ kubectl get deployment.extensions -n cloudflow
NAME                                READY   UP-TO-DATE   AVAILABLE   AGE
cloudflow-efs-efs-provisioner       1/1     1            1           2m51s
cloudflow-flink-flink-operator      1/1     1            1           2m40s
cloudflow-operator                  1/1     1            1           2m13s
cloudflow-sparkoperator             1/1     1            1           2m20s
cloudflow-strimzi-entity-operator   1/1     1            1           29s
strimzi-cluster-operator            1/1     1            1           2m28s```

Reply:
@skonto can you explain what effect this change will have? It is not clear from your comment.

Reply:
Right now we get an error:
```
Error from server (NotFound): deployments.extensions "cloudflow-sparkoperator-sparkoperator" not found
```
This is the fix you see above.

Reply:
Great find @skonto !